### PR TITLE
Fixes vampire screech emagging robots

### PR DESCRIPTION
**Pull request by Medoublej**
[BUG]
## About the PR 
Makes it so the vampire screech MK2 no longer Emags robots. Also fixes the MK2 so it now silences radios for 10 seconds.

## Why's this needed? 
It's a little strong to Emag a robot with a scream. Also the radio silencer did not function properly 

## Changelog

```
(u)Medoublej:
(+) Vampire's chiropteran screech MK2 now silences radios properly and no longer emags robots.
```

### CODE REVIEW

File: code/datums/abilities/vampire/screech.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	cooldown = 300
 	pointCost = 60
 	when_stunned = 1
+	var/duration = 50
```
Comment (ZeWaka) [inline code/datums/abilities/vampire/screech.dm:11]:
```suggestion
	var/duration = 50 SECONDS
```

Comment (ZeWaka) [reply code/datums/abilities/vampire/screech.dm:11]: bit too short imo

**Comment by comradef191** [comment]
> ```
> (u)Medoublej:
> (+) Vampire Scream MK2 now silences radios properly
> (+) Vampire Scream MK2 no longer enmags robots
> ```

Typo, Use Emag here instead-not Enmag...

**Comment by Medoublej** [comment]
Fixed it. Thanks for catching that!

**Review by ZeWaka — APPROVED** [review]

**Review by flrsh — APPROVED** [review]
Bit of a nitpick, but I would change your changelog to something like `(+)Vampire's chiropteran screech now silences radios properly and no longer emags robots`.

**Comment by Medoublej** [comment]
Done! Thanks for the feedback!

**Pull request merged**